server/src/printers/TestsPrinter.cpp

```diff
@@ -705,17 +705,33 @@ std::string printer::MultiLinePrinter::print(TestsPrinter *printer,
     std::stringstream structuredValuesWithPrefixes;
     structuredValuesWithPrefixes << "{" << NL;
     ++printer->tabsDepth;
+
+    int longestFieldIndexForUnionInit = view->getLongestFieldIndexForUnionInit();
```

Comment:
So now we are printing all the fields in union, but commenting all, except the longest.